samples/subsys/video/capture_to_lvgl/boards/mini_stm32h743.overlay

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+/ {
+	chosen {
+		zephyr,camera = &dcmi;
+	};
+};
+
+&pll {
+	div-m = <5>;
+	mul-n = <96>;
+	div-p = <2>;
+	div-q = <2>;
+	div-r = <2>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(240)>;
+	d1cpre = <1>;
+	hpre = <1>;
+	d1ppre = <2>;
+	d2ppre1 = <2>;
+	d2ppre2 = <2>;
+	d3ppre = <2>;
+};
+
+&i2c1 {
+	pinctrl-0 = <&i2c1_scl_pb8 &i2c1_sda_pb9 &rcc_mco_1_pa8>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	ov2640: ov2640@30 {
```

Comment:
what is the form factor / connector of the camera you're using? It feels like it would maybe be more reusable if you would be defining it as a shield?

Reply:
Please, see the image.

![image](https://github.com/user-attachments/assets/b56f731f-73d5-4763-b7f7-941e6f66488c)

Reply:
It looks like the connector of the ESP32-CAM modules made by various vendors.
https://timsblogplace.blogspot.com/2022/01/esp32-cam-rotating-camera.html

It looks like there are several cameras module proposed by WeAct Studio, that are all expected to fit into that connector. Some def-facto ESP32-CAM connector maybe?

I confirm that the FPC of OV7725 and OV2640 here both fit mechanically at least:
https://www.aliexpress.com/item/1005006632336183.html

Reply:
You are right! This board also supports the OV5640.

@kartben and @josuah, should I implement it as a shield?

Reply:
> should I implement it as a shield?

If you have the patience to, I think this would be appreciated!